ray-operator/config/samples/ray-service.high-availability.yaml

```diff
@@ -144,10 +139,10 @@ spec:
               resources:
                 limits:
                   cpu: 1
-                  memory: 1Gi
+                  memory: 2Gi
```

Comment:
Was removal of DAGDriver fixing the OOM issue or increasing the memory is the actual fix?

Reply:
Yes, I also tested with the YAML that removes the DAGDriver but still uses 1 GB of memory. I can still see the OOM error message by Ray dashboard.